releasenotes/notes/add-ipv6-support-for-hostnames-0fb700a1ef7ed57e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds support for IPv6 agent host names.
```

Comment:
Let's make to to specify that this is only for `DD_AGENT_HOST` if we're not adding support for `DD_TRACE_AGENT_URL`

Reply:
Sounds good. I'll do that.